client/me/get-apps/style.scss

```diff
@@ -1,53 +1,98 @@
-.get-apps__title {
-	font-size: 24px;
-	font-family: $serif;
-	font-weight: 700;
-	line-height: 32px;
-	margin-bottom: 8px;
+.get-apps__desktop,
+.get-apps__mobile {
+	margin-bottom: 0;
 
-	@include breakpoint( ">480px" ) {
-		margin-top: 24px;
+	@include breakpoint('>480px') {
+		display: flex;
+		flex-direction: row;
+		align-items: center;
 	}
-}
 
-.get-apps__subheading {
-	max-width: 500px;
-	margin: 16px auto;
+	h3 {
+		font-size: 15px;
+		margin-bottom: 4px;
+
+		@include breakpoint('>480px') {
+			font-size: 18px;
+		}
+	}
+
+	p {
+		font-size: 13px;
+		margin-bottom: 4px;
+
+		@include breakpoint('>480px') {
+			font-size: 15px;
+		}
+	}
+
+	p:last-child {
+		margin-bottom: 12px;
+
+		@include breakpoint('>480px') {
+			margin-bottom: 0;
+		}
+	}
 }
 
-.get-apps__app {
+.get-apps__illustration {
+	background-color: $white;
+	box-shadow: 0 0 0 1px transparentize( lighten( $gray, 20% ), .5 ),
+		0 1px 2px lighten( $gray, 30% );
+	color: $blue-dark;
+	max-width: 980px;
+	overflow: hidden;
+	padding: 0 24px 12px;
 	text-align: center;
-	float: left;
+
+	@include breakpoint('>480px') {
+		padding-bottom: 24px;
+	}
+
+	h2 {
```

Comment:
How many of these do we need to style? If possible we usually try to avoid a bare element selector if it's reasonable. The alternative here would be to pass add a classname like `get-apps__illustration-header` and unnest the rules here.